### ISSUE TITLE
Linux: application closes after clicking 'continue' on the first dialog.

### ISSUE THREAD

**Issue by willgfx** [DotBow/Blender-Launcher#12]
Upon launching, I get the "First, choose where Blender builds are stored" dialog.
After clicking 'continue', the application closes.

OS: Manjaro Linux (GNOME)
Blender Launcher version: v1.2.0

Console Output:
```
❯ ./Blender\ Launcher 
Gtk-Message: 11:40:38.815: Failed to load module "canberra-gtk-module"
Gtk-Message: 11:40:38.815: Failed to load module "canberra-gtk-module"

(Blender Launcher:11822): Gtk-WARNING **: 11:40:43.645: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Adwaita/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
Bail out! Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Adwaita/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
fish: “./Blender\ Launcher” terminated by signal SIGABRT (Abort)
```

**Comment by DotBow** [comment]
Hi!
Linux again giving another headache 😞 

Well, it totally on OS side, you probably need to install some module.
I'm very bad at Linux, so here what forums says:
> sudo apt-get install libcanberra-gtk-module

It there is a BL.log file, then send it to know exactly what's happened.
For now it is better to google for what's missing from GTK.

**Comment by willgfx** [comment]
I was able to build it locally and have it work, so you might be missing a dependency in your Linux build environment.

**Comment by DotBow** [comment]
Good to hear that managed to fix it!
But how I can know that there is something missing if only one user experiencing such problem... I can't even reproduce it, so there is no way to check anything.
I will try to wrap head around this problem, hope there is some way to have it fixed.

**Comment by willgfx** [comment]
I did some VM testing with various distributions:

- Ubuntu (GTK-based) - worked
- Pop! OS (GTK-based) - worked
- Solus Budgie (GTK-based) - same issue
- Manjaro KDE (QT-based) - worked
- Manjaro GNOME (QT-Based) - same issue

I tracked down the issue: https://0xacab.org/leap/bitmask-vpn/-/issues/77

It seems to be a bug with QT itself.
When QT is trying to display a GTK file dialog on a GTK-based desktop, it's looking for a library at absolute path which may differ by distribution and doesn't have any error handling. It doesn't have any issue if it's trying to display a QT file dialog. On my local build, it fell back to a QT file dialog when it was able to work.

The fix for anyone else with the issue:
`
sudo ln -sr /usr/lib /usr/lib/x86_64-linux-gnu
`
The path might be different depending on your distribution.

**Comment by DotBow** [comment]
Wow, thank you very much!
I think I can try to create symlink via code, it might be a good solution.
Really appreciate work you've done!

**Comment by DotBow** [comment]
So, for cross-platform support the best solution is using QT dialogs via QFileDialog.DontUseNativeDialog.
https://kb.froglogic.com/squish/qt/howto/automating-native-file-dialogs/
I don't need any fancy stuff for it anyway. Thanks again!

**Comment by willgfx** [comment]
Forcing it to use QT dialogs might be the best choice, so you don't have to implement any hacky workarounds.

**Comment by DotBow** [comment]
Resolved here 4840d3626b5c3dcb62122ce0b7be623df06501e1
Need some style fixes though.
